src/ilasm/asmman.cpp

```diff
@@ -965,8 +965,10 @@ HRESULT AsmMan::EmitManifest()
                 for(j=0; (hFile == INVALID_HANDLE_VALUE)&&(pwzInputFiles[j] != NULL); j++)
                 {
                     wcscpy_s(wzFileName,2048,pwzInputFiles[j]);
-                    pwz = wcsrchr(wzFileName,'\\');
+                    pwz = wcsrchr(wzFileName,DIRECTORY_SEPARATOR_CHAR_A);
+#ifndef FEATURE_PAL
```

Comment:
`#ifdef TARGET_UNIX` is a preferred way now, can you please change it here and at the two other places?

Reply:
Hi Jan, it looks like the TARGET_UNIX macro doesn't exist on the 3.1 branch, only on the mainline. FEATURE_PAL seemed to be the equivalent test on the older 3.1 codebase.

Reply:
Ah, I am sorry, I have not noticed this is the 3.1 port and not the original 5.0 change.